### PR TITLE
Display recent withdrawal information to console

### PR DESCRIPTION
**Pull request by quantumagi**
Shows information relating to recent withdrawals in the console.

### CODE REVIEW

File: src/Stratis.FederatedPeg.Features.FederationGateway/TargetChain/Withdrawal.cs [hunk 1]
```diff
@@ -38,5 +38,16 @@ public override string ToString()
...
+
+        public string GetInfo()
+        {
+            return string.Format("Tran#={0} Dep#={1} Amount={2,12} Addr={3} BlkNum={4,8} BlkHash={5}",
+                this.Id.ToString().Substring(0, 6),
+                this.DepositId.ToString().Substring(0, 6),
+                this.Amount.ToString(),
+                this.TargetAddress,
+                this.BlockNumber,
+                this.BlockHash.ToString().Substring(0, 6));
```
Comment (bokobza) [inline src/Stratis.FederatedPeg.Features.FederationGateway/TargetChain/Withdrawal.cs]: I think blockHash may be null is the transaction hasn't been confirmed yet.

File: src/Stratis.FederatedPeg.Features.FederationGateway/FederationGatewayFeature.cs [hunk 2]
```diff
@@ -203,8 +208,18 @@ private void AddComponentStats(StringBuilder benchLog)
                                 + " Confirmed balance: " + balances.ConfirmedAmount.ToString().PadRight(LoggingConfiguration.ColumnLength)
                                 + " Unconfirmed balance: " + balances.UnConfirmedAmount.ToString().PadRight(LoggingConfiguration.ColumnLength)
                                 + " Federation Status: " + (this.federationWalletManager.IsFederationActive() ? "Active" : "Inactive"));
-            benchLog.AppendLine();
+            benchLog.AppendLine("-- Recent Withdrawals --");
```
Comment (bokobza) [inline src/Stratis.FederatedPeg.Features.FederationGateway/FederationGatewayFeature.cs]: Can we have an extra line above this?

File: src/Stratis.FederatedPeg.Features.FederationGateway/Wallet/FederationWalletManager.cs [hunk 3]
```diff
@@ -811,6 +811,20 @@ public bool RemoveTransientTransactions(uint256 depositId = null)
             }
         }
 
+        /// <inheritdoc />
+        public IEnumerable<(Transaction, TransactionData, IWithdrawal)> EnumWithdrawals()
```
[the remark this answers is not in the document]
Comment (quantumagi) [reply src/Stratis.FederatedPeg.Features.FederationGateway/Wallet/FederationWalletManager.cs]: Done.

File: src/Stratis.FederatedPeg.Features.FederationGateway/Wallet/FederationWalletManager.cs [hunk 4]
```diff
@@ -811,6 +811,20 @@ public bool RemoveTransientTransactions(uint256 depositId = null)
...
+        {
+            foreach (TransactionData transactionData in this.Wallet.MultiSigAddress.Transactions.OrderByDescending(t => t.CreationTime))
+            {
+                Transaction walletTran = transactionData.GetFullTransaction(this.network);
```
[the remark this answers is not in the document]
Comment (quantumagi) [reply src/Stratis.FederatedPeg.Features.FederationGateway/Wallet/FederationWalletManager.cs]: Done.

File: src/Stratis.FederatedPeg.Features.FederationGateway/TargetChain/Withdrawal.cs (hunk 1 above)
Comment (quantumagi) [reply src/Stratis.FederatedPeg.Features.FederationGateway/TargetChain/Withdrawal.cs]: Done.

File: src/Stratis.FederatedPeg.Features.FederationGateway/FederationGatewayFeature.cs (hunk 2 above)
Comment (quantumagi) [reply src/Stratis.FederatedPeg.Features.FederationGateway/FederationGatewayFeature.cs]: Done.

**Pull request merged**